transport_nantes/photo/views.py

```diff
@@ -21,6 +21,10 @@ class UploadEntry(LoginRequiredMixin, CreateView):
 
     success_url = reverse_lazy('photo:confirmation')
 
+    def get(self, request, *args, **kwargs):
+        logger.info(f"{request.user} landed on photo contest page")
```

Comment:
Careful, this is vague.  When reading thousands of lines of logs, will you really know what this means?  Will another dev who hasn't worked on this?  What I mean is that your language in the log message is about the user experience and not about the code.  So it's not enough to know the code, you need to know, for example, that we use this code for a photo competition, and, morever, at some future date when we use it for something totally different (as always happens with the code that is not consigned to the dust bin), that we _used_ to use it for a photo competition.

The log message includes the module and line of code that emits it now, and this is really key to understanding logs and tracing back to code.

Something more functional makes more sense to me:
```f"UploadEntry.get() from {request.user}"```